src/components/Carousel/Carousel.js

```diff
@@ -111,18 +132,41 @@ class Carousel extends Component {
     });
   }
 
+  renderChildren() {
+    const { children, items } = this.props;
+    const { widthOfSlide } = this.state;
+    const style = { maxWidth: `${widthOfSlide}px` };
+    const listOfChildren = [...children];
+
+    return listOfChildren.map((item, i) => (
+      <div
+        key={items[i].id}
+        className={`${CN}__child`}
+        style={style}
+      >
+        {item}
+      </div>
+    ));
+  }
+
+
   render() {
-    const { children } = this.props;
     const { allTheWayLeft, allTheWayRight } = this.state;
     const navClasses = cx({ [`${CN}__nav`]: true });
-    const leftNavClasses = cx({
-      [`${CN}__left-nav`]: true,
-      [`${CN}__nav-disabled`]: allTheWayLeft
-    }, navClasses);
-    const rightNavClasses = cx({
-      [`${CN}__right-nav`]: true,
-      [`${CN}__nav-disabled`]: allTheWayRight
-    }, navClasses);
+    const leftNavClasses = cx(
+      {
+        [`${CN}__left-nav`]: true,
+        [`${CN}__nav-disabled`]: allTheWayLeft
+      },
+      navClasses
+    );
+    const rightNavClasses = cx(
+      {
+        [`${CN}__right-nav`]: true,
```

Comment:
Maybe you can move this class out of object if it is always true?

````
const rightNavClasses = cx({
        [`${CN}__nav-disabled`]: allTheWayRight
      },
`${CN}__right-nav`,
      navClasses
    );

Reply:
moving this class out of object is making this button disabled constantly (just checked it)

at this moment it reflects on position alTheWayLeft of allTheWayRight.